client/package.json

```diff
@@ -8,38 +8,53 @@
     "client/node_modules"
   ],
   "directories": {
-    "test": "test"
+    "test": "tests"
   },
   "dependencies": {
+    "html-loader": "^0.4.4",
+    "install": "^0.8.1",
+    "react-dropdown": "^1.1.0",
     "react-on-rails": "6.1.1"
   },
   "devDependencies": {
+    "babel": "6.5.2",
     "babel-cli": "^6.16.0",
     "babel-core": "^6.17.0",
     "babel-loader": "^6.2.5",
     "babel-polyfill": "^6.7.4",
     "babel-preset-es2015": "^6.16.0",
+    "babel-preset-latest": "6.16.0",
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-0": "^6.16.0",
     "babel-runtime": "^6.6.1",
+    "cheerio": "^0.22.0",
+    "enzyme": "^2.4.1",
     "es5-shim": "^4.5.7",
     "expose-loader": "^0.7.1",
     "imports-loader": "^0.6.5",
     "jasmine": "^2.5.2",
     "jasmine-core": "^2.5.2",
+    "json-loader": "^0.5.4",
     "karma": "^1.3.0",
+    "karma-babel-preprocessor": "^6.0.1",
+    "karma-chrome-launcher": "^0.2.3",
+    "karma-coverage": "^0.4.2",
     "karma-jasmine": "^1.0.2",
+    "karma-jquery": "^0.1.0",
     "karma-sourcemap-loader": "^0.3.7",
+    "karma-spec-reporter": "0.0.20",
     "karma-webpack": "^1.8.0",
+    "lodash": "^4.13.1",
```

Comment:
Lodash is a very nice library. But how did it get here with this commit? Is this a karma dependency?